cxx-sensors/src/main/java/org/sonar/cxx/sensors/drmemory/CxxDrMemorySensor.java

```diff
@@ -85,15 +91,6 @@ protected void processReport(final SensorContext context, File report) {
           }
         }
       }
-    } catch (IOException e) {
```

Comment:
@Bertk why is this removed? Should break or continue (handling inside validateRecovery).